ksql-common/src/main/java/io/confluent/ksql/util/WelcomeMsgUtils.java

```diff
@@ -50,7 +50,7 @@ public static void displayWelcomeMessage(
         "==========================================="
     };
 
-    final String copyrightMsg = "Copyright 2017 Confluent Inc.";
+    final String copyrightMsg = "Copyright 2018 Confluent Inc.";
```

Comment:
I believa @blueedgenick mentioned that this should probably be "Copyright 2017-2018 ..."

Reply:
As Damian says, @blueedgenick requested I revert my change when I made the same change:

https://github.com/confluentinc/ksql/pull/806#issuecomment-368953235

Reply:
Ah.. Yes. It looks like that's the practice for most products, like the copyright notice on my Mac: 
![screen shot 2018-03-07 at 11 25 13 am](https://user-images.githubusercontent.com/830220/37113449-4085f67e-21fa-11e8-8b21-43abc6c109da.png)